scheduler/db/scheduler.changelog-master.yaml

```diff
@@ -134,6 +139,11 @@ databaseChangeLog:
                   type: integer
                   constraints:
                     nullable: true
+              - column:
+                  name: guid
+                  type: varchar(50)
+                  constraints:
+                    nullable: false
```

Comment:
Yep. I'm aware of that. But that was the case when we didn't have any deployment and the idea was to not create any changesets till we have done the first release. But recently we have already deployed a version internally and hence would like to freeze on the current changelog as is and have new changesets for db updates.
@julz your comments?